Tools/clinic/clinic.py

```diff
@@ -884,6 +884,7 @@ def parser_body(
                 displayname = parameters[0].get_displayname(0)
                 parsearg = converters[0].parse_arg(argname, displayname, limited_capi=limited_capi)
                 if parsearg is None:
+                    converters[0].use_converter()
```

Comment:
Is this needed? If I remove the line there is no change if I regen clinic.

Reply:
```console
$ git grep "\<PyArg_Parse\>" **/clinic/*
Modules/clinic/_cursesmodule.c.h:    if (!PyArg_Parse(arg, "y:putp", &string)) {
Modules/clinic/_winapi.c.h:    if (!PyArg_Parse(arg, "" F_HANDLE ":CloseHandle", &handle)) {
Modules/clinic/_winapi.c.h:    if (!PyArg_Parse(arg, "I:ExitProcess", &ExitCode)) {
Modules/clinic/_winapi.c.h:    if (!PyArg_Parse(arg, "" F_HANDLE ":GetExitCodeProcess", &process)) {
Modules/clinic/_winapi.c.h:    if (!PyArg_Parse(arg, "" F_HANDLE ":GetModuleFileName", &module_handle)) {
Modules/clinic/_winapi.c.h:    if (!PyArg_Parse(arg, "k:GetStdHandle", &std_handle)) {
Modules/clinic/_winapi.c.h:    if (!PyArg_Parse(arg, "" F_POINTER ":UnmapViewOfFile", &address)) {
Modules/clinic/_winapi.c.h:    if (!PyArg_Parse(arg, "" F_POINTER ":VirtualQuerySize", &address)) {
Modules/clinic/posixmodule.c.h:    if (!PyArg_Parse(arg, "" _Py_PARSE_PID ":sched_getscheduler", &pid)) {
Modules/clinic/posixmodule.c.h:    if (!PyArg_Parse(arg, "" _Py_PARSE_PID ":sched_getparam", &pid)) {
Modules/clinic/posixmodule.c.h:    if (!PyArg_Parse(arg, "" _Py_PARSE_PID ":sched_rr_get_interval", &pid)) {
Modules/clinic/posixmodule.c.h:    if (!PyArg_Parse(arg, "" _Py_PARSE_PID ":sched_getaffinity", &pid)) {
Modules/clinic/posixmodule.c.h:    if (!PyArg_Parse(arg, "" _Py_PARSE_PID ":getsid", &pid)) {
Modules/clinic/posixmodule.c.h:    if (!PyArg_Parse(arg, "" _Py_PARSE_INTPTR ":get_handle_inheritable", &handle)) {
Modules/clinic/unicodedata.c.h:    if (!PyArg_Parse(arg, "s#:lookup", &name, &name_length)) {
Tools/clinic/clinic.py:                        if (!PyArg_Parse(%s, "{format_units}:{name}", {parse_arguments})) {{
```

Reply:
`use_converter()` is more for converters using `O&` + callback function. I don't think that currently, the Python code base goes to this code path with a `O&` converter which emits an `#include`.

`use_converter()` is mainly needed to generate `#include`.

Reply:
> `use_converter()` is mainly needed to generate `#include`.

IMO, it would be better if this was done implicitly rather than explicitly.

Reply:
That would be nice, but I failed to find a way to distinguish when code to parse arguments is "inlined" vs `O&` converters. The `use_converter()` method is called explicitly, but it only has to be called in a few places.

This PR removed an unused `#include` in `Modules/clinic/_ssl.c.h`.